app-frontend/src/app/components/pages/project/createAnalysis/index.html

```diff
@@ -1,71 +1,80 @@
-        <item-selector>
-          <rf-list-item-selector
-              id="layer.id"
-              selected="$ctrl.isSelected(layer.id)"
-              on-select="$ctrl.onSelect(id)"
-              color="layer.colorGroupHex"
-          ></rf-list-item-selector></item-selector>
-        <item-title><strong ng-attr-title="{{layer.name}}">{{layer.name}}</strong></item-title>
-        <item-date>{{layer.date | date}}</item-date>
-        <item-actions>
-          <rf-list-item-actions
-              actions="$ctrl.layerActions.get(layer.id)" >
-          </rf-list-item-actions>
-        </item-actions>
-      </rf-list-item>
+        <rf-list-item ng-repeat="layer in $ctrl.layerList track by layer.id">
+            <item-selector>
+                <rf-list-item-selector
+                    id="layer.id"
+                    selected="$ctrl.isSelected(layer.id)"
+                    on-select="$ctrl.onSelect(id)"
+                    color="layer.colorGroupHex"
+                ></rf-list-item-selector
+            ></item-selector>
+            <item-title>
+                <strong ng-attr-title="{{ layer.name }}">{{ layer.name }}</strong>
+            </item-title>
+            <item-date ng-if="layer.rangeStart && layer.rangeEnd">
+                {{ layer.rangeStart | date }} - {{ layer.rangeEnd | date }}
+            </item-date>
+            <item-actions>
+                <rf-list-item-actions actions="$ctrl.layerActions.get(layer.id)">
+                </rf-list-item-actions>
+            </item-actions>
+        </rf-list-item>
+    </div>
+</div>
+<div class="main page-content-container">
+    <div class="page-header wide">
+        <h3>Choose an analysis to run</h3>
     </div>
-  </div>
-  <div class="container fit-height column-stretch analysis-list">
-    <div class="row content stack-sm">
-      <div class="column-12">
-        <div class="dashboard-header">
-          <h3>Choose an analysis to run</h3>
-          <div class="flex-fill"></div>
-          <rf-search on-search="$ctrl.fetchTemplates(1, value)"  value="$ctrl.search"  placeholder="Search for analysis" auto-focus="false">
-          </rf-search>
-          <select
-              class="form-control"
-              ng-model="$ctrl.currentOwnershipFilter"
-              ng-change="$ctrl.handleOwnershipFilterChange()"
-          >
-            <option value="">All</option>
-            <option value="owned">Owned by me</option>
-            <option value="shared">Shared with me</option>
-          </select>
+    <div class="page-card transparent no-padding wide">
+        <div class="flex-display input-group">
+            <div class="help-text flex-fill">
+                The list of analyses has been filtered down to include only single-source analyses.
+            </div>
+            <rf-search
+                on-search="$ctrl.fetchTemplates(1, value)"
+                value="$ctrl.search"
+                placeholder="Search for analysis"
```

Comment:
If we were to change the above wording to `template`, I think we would want to change here too.